### PR TITLE
Miscellaneous ScanCode-related minor improvements

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: scanner/src/test/assets/mime-types-2.1.18_scancode-2.9.7.json [hunk 1]
```diff
@@ -0,0 +1,506 @@
+{
+  "scancode_notice": "Generated with ScanCode and provided on an \"AS IS\" BASIS, WITHOUT WARRANTIES\nOR CONDITIONS OF ANY KIND, either express or implied. No content created from\nScanCode should be considered or used as legal advice. Consult an Attorney\nfor any legal advice.\nScanCode is a free software code scanning tool from nexB Inc. and others.\nVisit https://github.com/nexB/scancode-toolkit/ for support and download.",
+  "scancode_version": "2.9.7",
+  "scancode_options": {
+    "input": "/home/sse1be/Development/mime-types",
```
Comment (mnonnenmacher) [inline scanner/src/test/assets/mime-types-2.1.18_scancode-2.9.7.json]: You might want to replace your home folder with a generic path.

File: scanner/src/test/assets/mime-types-2.1.18_scancode-2.9.7.json [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+    "--copyright": true,
+    "--ignore": [
+      "*.ort.yml",
+      "HERE_NOTICE",
+      "META-INF/DEPENDENCIES"
+    ],
+    "--info": true,
+    "--json-pp": "/home/sse1be/Development/oss-review-toolkit/cli/out/scanner/native-scan-results/mime-types_ScanCode.json",
```
Comment (mnonnenmacher) [inline scanner/src/test/assets/mime-types-2.1.18_scancode-2.9.7.json]: You might want to replace your home folder with a generic path.

File: scanner/src/test/assets/mime-types-2.1.18_scancode-3.0.2.json [hunk 3]
```diff
@@ -0,0 +1,516 @@
+{
+  "headers": [
+    {
+      "tool_name": "scancode-toolkit",
+      "tool_version": "3.0.2",
+      "options": {
+        "input": "/home/sse1be/Development/mime-types",
```
Comment (mnonnenmacher) [inline scanner/src/test/assets/mime-types-2.1.18_scancode-3.0.2.json]: You might want to replace your home folder with a generic path.

File: scanner/src/test/assets/mime-types-2.1.18_scancode-3.0.2.json [hunk 4]
```diff
@@ -0,0 +1,516 @@
...
+        "--copyright": true,
+        "--ignore": [
+          "*.ort.yml",
+          "HERE_NOTICE",
+          "META-INF/DEPENDENCIES"
+        ],
+        "--info": true,
+        "--json-pp": "/home/sse1be/Development/oss-review-toolkit/cli/out/scanner/native-scan-results/mime-types_ScanCode.json",
```
Comment (mnonnenmacher) [inline scanner/src/test/assets/mime-types-2.1.18_scancode-3.0.2.json]: You might want to replace your home folder with a generic path.

File: scanner/src/test/kotlin/ScanCodeTest.kt [hunk 5]
```diff
@@ -39,6 +39,26 @@ import java.time.Instant
 class ScanCodeTest : WordSpec({
     val scanner = ScanCode("ScanCode", ScannerConfiguration())
 
+    "ScanCode 2 results" should {
+        "be correctly summarized" {
+            val resultFile = File("src/test/assets/mime-types-2.1.18_scancode-2.9.7.json")
+            val result = scanner.getRawResult(resultFile)
+            val summary = scanner.generateSummary(Instant.now(), Instant.now(), result)
+
+            summary.fileCount shouldBe 10
```
Comment (mnonnenmacher) [inline scanner/src/test/kotlin/ScanCodeTest.kt]: Why don't you also verify the license findings?

File: scanner/src/test/kotlin/ScanCodeTest.kt [hunk 6]
```diff
@@ -39,6 +39,26 @@ import java.time.Instant
...
+        }
+    }
+
+    "ScanCode 3 results" should {
+        "be correctly summarized" {
+            val resultFile = File("src/test/assets/mime-types-2.1.18_scancode-3.0.2.json")
+            val result = scanner.getRawResult(resultFile)
+            val summary = scanner.generateSummary(Instant.now(), Instant.now(), result)
+
+            summary.fileCount shouldBe 10
```
Comment (mnonnenmacher) [inline scanner/src/test/kotlin/ScanCodeTest.kt]: Why don't you also verify the license findings?

File: scanner/src/test/kotlin/ScanCodeTest.kt (hunk 5 above)
Comment (sschuberth) [reply scanner/src/test/kotlin/ScanCodeTest.kt]: That's already done in other tests. This test is mainly to ensure the different ways to get the file count in ScanCode 2 vs. ScanCode 3 results work, and that generally no exception is thrown.

File: scanner/src/test/kotlin/ScanCodeTest.kt (hunk 6 above)
Comment (sschuberth) [reply scanner/src/test/kotlin/ScanCodeTest.kt]: See my previous comment.

File: scanner/src/test/assets/mime-types-2.1.18_scancode-2.9.7.json (hunk 1 above)
Comment (sschuberth) [reply scanner/src/test/assets/mime-types-2.1.18_scancode-2.9.7.json]: Thanks for pointing that out!

**Pull request merged**